app/models/invoice.rb

```diff
@@ -4,4 +4,8 @@ class Invoice < ApplicationRecord
   has_many :transactions
   has_many :invoice_items
   has_many :items, through: :invoice_items
+
+  def total_revenue
+    invoice_items.sum(:total_revenue)
```

Comment:
do we have a :total_revenue method on invoice items ? If not, we can move this there, and then add the merchant relationship to invoice items, perhaps